flocker/node/test/test_deploy.py

```diff
@@ -1038,7 +1041,7 @@ def test_discover_manifestation_with_size(self):
         )
         d = api.discover_state(self.EMPTY_NODESTATE)
 
-        self.assertItemsEqual(
```

Comment:
How do the other changes (`RestartNever`, `self.DATASET_ID`) relate to the `PClass` change?